pydantic/_internal/_model_construction.py

```diff
@@ -30,8 +30,7 @@
 
 __all__ = 'object_setattr', 'init_private_attributes', 'inspect_namespace', 'complete_model_class', 'MockValidator'
 
-
-IGNORED_TYPES: tuple[Any, ...] = (FunctionType, property, type, classmethod, staticmethod)
+IGNORED_TYPES: tuple[Any, ...] = (FunctionType, property, type, classmethod, staticmethod, PydanticDecoratorMarker)
```

Comment:
What’s the difference between this and PYDANTIC_NON_FIELD_TYPES?

Reply:
can we use `ignored_types` instead of `non_field_types`?

Reply:
looks like I added `PYDANTIC_NON_FIELD_TYPES` then refactored things to stop using it; I will drop it.

I will also change `non_field_types` to `ignored_types`.